lambdas/retrieve-cri-credential/src/main/java/uk/gov/di/ipv/core/retrievecricredential/RetrieveCriCredentialHandler.java

```diff
@@ -204,6 +207,86 @@ public Map<String, Object> handleRequest(Map<String, String> input, Context cont
         }
     }
 
+    private Map<String, Object> processPendingResponse(
```

Comment:
Does this require an audit event?

Reply:
It may well do, but I've added that as a separate subtask (https://govukverify.atlassian.net/browse/PYIC-2911) which may well get pushed to the fast-follow epic